### ISSUE TITLE
Instructions for use of WSS matchmaking URL

### ISSUE THREAD

**Issue by kalidrout** [Lawin0129/LawinServerV2#133]
Hello, I've set up the server sucessfully and i can log in but i am facing an issue that i cannot ready up. I believe i shall use something else (like an command/an injector) but i havent found out anything. Could someone help me with getting me into matches. Much appreciated

**Comment by hagiboykisser** [comment]
u gottta use fortmatchamker

**Comment by H0NEYCUT** [comment]
Note: Fortmatchmaker does not work on c1s0,c1s1, and c1s2. As far as I know, the earliest supported season is c1s3. Best to stick to the last release of c1s3 to be sure, which works great with the matchmaker on my end. Version 3.5 that is.

**Comment by kalidrout** [comment]
is there a different matchmaker that supports s1 c1 to sX c1 as i wanna play those OG seasons. I had success with using lawin with reboot launcher gameserver

**Comment by H0NEYCUT** [comment]
I do not believe there is for the first few seasons. Season 3 truly is the oldest one with a working matchmaker.
The reason for this, I don't know. Sorry !

**Comment by kalidrout** [comment]
i appreciate it. Imma hold this issue open if someone else knows

**Comment by kalidrout** [comment]
no i think u don’t get it. I attempt to play on season 4-7 chapter 1 not older than that

**Comment by H0NEYCUT** [comment]
> Is there a matchmaker that supports s1 c1 to sX, as I wanna play **those** seasons.
Because of this line I assumed you were also interested in playing season 0 to 2. 

If you're only gonna play season 4-7, then use fortmatchmaker. Those work flawlessly.

**Comment by kalidrout** [comment]
Do you know any cli based gameservers? I used reboot but i need an separate account for the gameserver and i do not know how to set it

**Comment by H0NEYCUT** [comment]
I just used an alt account on discord to create the profile for my server. 
First I launch the server with this account, then in the reboot client I change my credentials and launch fn regularly.

The gameserver works by injecting the server software into the game itself, so no, I dont think there's a cli. Reboot is the best open-source we've got right now.

**Comment by kalidrout** [comment]
How do i set the generated account to reboot. Thats my main issue

**Comment by H0NEYCUT** [comment]
Be sure you're running the latest launcher by Auties (Check the reboot discord)

In the top left, press the user icon, and insert your generated server account's credentials.

Go to the tab named "server" and press "start hosting". When the server has launched, press the user icon in Reboot again, then insert your player's credentials.
Now, go back to the tab named "play", and press play, then play again on the next popup.

**Comment by kalidrout** [comment]
thanks so much bro. You have saved me hours

**Comment by xy8q** [comment]
From my previous experience, Season 4 had some issues with authorization. The first season that worked flawlessly for me was Season 5. I remember there was a fix for the auth problem in Season 4, if I find it, I will let you know.

**Comment by xy8q** [comment]
I think its just an issue with Season 4, not sure about auth problem on Season 3.
